src/server/scripts/Kalimdor/Firelands/boss_ragnaros_firelands.cpp

```diff
@@ -2415,10 +2428,11 @@ class npc_ragnaros_sulfuras : public CreatureScript
             void IsSummonedBy(Unit* /*summoner*/) override
             {
                 me->CastSpell(me, SPELL_SULFURAS, true);
-                if (InstanceScript* instance = me->GetInstanceScript())
-                    if (Creature* ragnaros = ObjectAccessor::GetCreature(*me, instance->GetGuidData(NPC_RAGNAROS)))
-                        if (ragnaros->IsAIEnabled)
-                            ragnaros->AI()->JustSummoned(me);
+            }
+
+            void UpdateAI(uint32 diff) override
+            {
+                me->Say("I'm still here", LANG_UNIVERSAL);
```

Comment:
It would be preferable to get text from the DB.
Check the tables `hotfixes`.`broadcast_text` and
`world`.`creature_text` for existing Say text.